server/routes/v1/article.js

```diff
@@ -1,8 +1,10 @@
 import express from 'express';
+import articles from '<controllers>/article';
+import verifyUser from '<middlewares>/verifyUser';
 import { validateCreateArticle } from '<validations>/article';
 
 const article = express.Router();
 
-article.post('/', validateCreateArticle);
+article.post('/', validateCreateArticle, verifyUser, articles);
```

Comment:
I think this should not be verifyUser because the middleware is not verifying the user its actually decoding the token that is to be used in your route